resources/sts/4.12/hypershift/ControlPlaneOperatorPolicy.json

```diff
@@ -4,11 +4,66 @@
         {
             "Action": [
                 "ec2:DescribeVpcEndpoints",
+                "ec2:DescribeVpcs",
+                "ec2:DescribeSecurityGroups",
                 "route53:ListHostedZones"
             ],
             "Effect": "Allow",
             "Resource": "*"
         },
+        {
+            "Sid": "CreateSecurityGroups",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:CreateSecurityGroup"
+            ],
+            "Resource": [
+                "arn:aws:ec2:*:*:security-group*/*"
+            ],
+            "Condition": {
+                "StringEquals": {
+                    "aws:RequestTag/red-hat-managed": "true"
+                }
+            }
+        },
+        {
+            "Sid": "DeleteSecurityGroup",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DeleteSecurityGroup"
+            ],
+            "Resource": [
+                "arn:aws:ec2:*:*:security-group*/*"
+            ],
+            "Condition": {
+                "StringEquals": {
+                    "aws:ResourceTag/red-hat-managed": "true"
+                }
+            }
+        },
+        {
+            "Sid": "SecurityGroupIngressEgress",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:AuthorizeSecurityGroupIngress",
+                "ec2:AuthorizeSecurityGroupEgress",
+                "ec2:RevokeSecurityGroupIngress",
+                "ec2:RevokeSecurityGroupEgress"
+            ],
+            "Resource": [
+                "arn:aws:ec2:*:*:security-group*/*"
+            ]
+        },
```

Comment:
I dont think it has an "impact" as all those actions only work on security groups and security group rules. but it is best practice to avoid using '*' resource where possible. Makes it more readable too IMO.

Reply:
Got it, thanks!